src/runtimes/node/bundlers/nft/index.ts

```diff
@@ -85,6 +86,31 @@ const getIgnoreFunction = (config: FunctionConfig) => {
   }
 }
 
+// Returns the module format that should be used when transpiling a TypeScript
+// file.
```

Comment:
NIT: comments for functions/methods or variables should always be JSDoc as the IntelliSense can pick them up. A regular code block comment should be just used inside functions as it is not part of the AST

```suggestion
/** 
 * Returns the module format that should be used when transpiling a TypeScript
 * file.
 */
```

Reply:
Done in https://github.com/netlify/zip-it-and-ship-it/pull/1580/commits/99f27bafdce87d3667f82c7619a9f1a7f92f84b1.